src/test/scala/nl.knaw.dans.easy.deposit/servlets/UploadSpec.scala

```diff
@@ -192,6 +192,54 @@ class UploadSpec extends DepositServletFixture {
       ).map("data/path/to/dir/" + _)
 
     }
+    // get should show uploaded files
+    get(
+      uri = s"/deposit/$uuid/file/",
+      headers = Seq(fooBarBasicAuthHeader),
+    ) {
+      status shouldBe OK_200
+      body should include ("""{"filename":"readme.md","dirpath":"path/to/dir",""")
+      body should include ("""{"filename":"._login.html","dirpath":"path/to/dir/__MACOSX",""")
```

Comment:
I would not expect the `__MACOSX` directory (and its files) to be in this listing. Instead I would expect them to not even be saved in the bag, while extracting the zip. These files are used for the encoding process only.

Reply:
@lindareijnhoudt @janvanmansum 
Didn't we decide in the past to store store whatever is stored in a zip file?
Might depend though on whether these files are created by the OS or by the zip-process.

Reply:
When using the command line `unzip` the `__MACOSX` directory gets extracted.

Reply:
When zipping with the command line `zip` we don't get extras.

Reply:
If the ui zips, it shouldn't use the mac zipper. If the depositor zipped we might not be sure it was zipped by mac or even a mix.

Reply:
UI doesn't zip; the user does. Same for ever other client that might use this api in the future.